trip-ninja-docs/src/common/GeneralConstants.tsx

```diff
@@ -0,0 +1,7 @@
+export enum NavLinkUrls {
+    HOME = "/",
+    FARESTRUCTURE = "/farestructure/overview/",
+    VI = "/virtual-interlining/overview",
```

Comment:
We should probably be consistent on whether or not the urls end in a slash, any thoughts?

Reply:
ouf yep, good catch! updated!